[2022-07-28 15:09:43,360] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: QCOM_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:43,469] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: QCOM_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:43,470] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:43,470] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-28 15:09:43,474] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:43,536] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-28 14:30:00+00:00
[2022-07-28 15:09:43,546] {standard_task_runner.py:52} INFO - Started process 817 to run task
[2022-07-28 15:09:43,554] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'QCOM_historic_data', 'create_historic_table', 'scheduled__2022-07-28T14:30:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmp0ayu_bmm', '--error-file', '/tmp/tmpye01f65p']
[2022-07-28 15:09:43,567] {standard_task_runner.py:80} INFO - Job 105: Subtask create_historic_table
[2022-07-28 15:09:43,930] {task_command.py:371} INFO - Running <TaskInstance: QCOM_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [running]> on host a2299eb83309
[2022-07-28 15:09:44,961] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=QCOM_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T14:30:00+00:00
[2022-07-28 15:09:45,048] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-28 15:09:45,120] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__QCOM_ (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            symbol VARCHAR(32) NOT NULL,
            PRIMARY KEY (symbol, datetime)
        )
    , parameters: None
[2022-07-28 15:09:45,190] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 15:09:45,432] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=QCOM_historic_data, task_id=create_historic_table, execution_date=20220728T143000, start_date=20220728T150943, end_date=20220728T150945
[2022-07-28 15:09:45,497] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 15:09:46,068] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
